### PR TITLE
supports topicNameStrategy, topicRecordNameStrategy and recordNameStr…

### PR DESCRIPTION
**Pull request by afirouzabady**
add new argument `subjectNameStrategy` to set one of the following strategies:

- `topicNameStrategy`
- `topicRecordNameStrategy`
- `recordNameStrategy`

### CODE REVIEW

**Comment by danhawkins** [comment]
+1  for this

**Comment by nicovogelaar** [comment]
Thank you for your contribution!

File: src/main/java/com/nicovogelaar/kafka/mirrormaker/SchemaRegistryTransfer.java [hunk 1]
```diff
@@ -50,8 +52,16 @@ public SchemaRegistryTransfer(String args) {
         targetSchemaRegistryClient = new CachedSchemaRegistryClient(arguments.targetUrl, schemaCapacity, configs);
 
         schemaCache = new SynchronizedCache<>(new LRUCache<>(schemaCapacity));
-
-        subjectNameStrategy = new TopicNameStrategy();
+        if(arguments.subjectNameStrategy.equals("topicNameStrategy")){
```
Comment (nicovogelaar) [inline src/main/java/com/nicovogelaar/kafka/mirrormaker/SchemaRegistryTransfer.java]: Can you please re-format the code?

```java
        if (arguments.subjectNameStrategy.equals("topicNameStrategy")) {
            subjectNameStrategy = new TopicNameStrategy();
        } else if (arguments.subjectNameStrategy.equals("topicRecordNameStrategy")) {
            subjectNameStrategy = new TopicRecordNameStrategy();
        } else if (arguments.subjectNameStrategy.equals("recordNameStrategy")) {
            subjectNameStrategy = new RecordNameStrategy();
        } else {
            subjectNameStrategy = new TopicNameStrategy();
        }
```

**Comment by nicovogelaar** [comment]
Can you please add the new argument `subjectNameStrategy` to the Usage on the README?

**Pull request merged**